v2.1/export.md

```diff
@@ -0,0 +1,159 @@
+---
+title: EXPORT
+summary: Export tabular data from CockroachDB cluster in CSV format.
+toc: false
+---
+
+The `EXPORT` [statement](sql-statements.html) exports tabular data to CSV files.
+
+{{site.data.alerts.callout_danger}}The <code>EXPORT</code> feature is only available to <a href="https://www.cockroachlabs.com/product/cockroachdb/">enterprise</a> users. Also note that this feature is currently under development and is slated for full release in CockroachDB 2.1. The feature flags and behavior are subject to change. {{site.data.alerts.end}}
```

Comment:
Should we mention `--format=csv > myfile.csv` in the note about enterprise? If so, should we just have that example moved to its own section ### Non-Distributed Export Using the SQL Shell that can be linked to by the enterprise and known-limitation sections?